app/src/main/java/com/randomappsinc/studentpicker/listpage/ListActivity.java

```diff
@@ -21,6 +21,9 @@
 
 public class ListActivity extends StandardActivity implements ShakeDetector.Listener {
 
+    public static final int EDIT_NAME_LIST_FRAGMENT = 1;
```

Comment:
nit: Removing fragment from the name hides implementation, so I think this is slightly cleaner.
```suggestion
    public static final int EDIT_NAME_LIST_PAGE_POSITION = 1;
```